### ISSUE TITLE
UPS-Lite V1.2 - Replacing the original battery with an 18650 battery

### ISSUE THREAD

**Issue by xt2six** [linshuqin329/UPS-Lite#15]
Good morning,

I have a problem with the original battery of assembly 1.2. Before starting the repair, I would like to have your opinion if I should keep the circuit soldered to the original battery (HLN 1832-6 - chips KIA SEMICONDUCTORS - 8205a) to place it on a 1 x 18650 battery holder?

What amp capacity do you recommend for 18650 batteries? (1,200?)

![img_20240509_115825](https://github.com/linshuqin329/UPS-Lite/assets/88111493/760e84d5-ebc4-4762-9325-443eac9171c9)

![S0aa9ee13d9a540c48c55176adcac9e27Y](https://github.com/linshuqin329/UPS-Lite/assets/88111493/9a58d25d-27a6-4404-b07f-85499e27232d)

![img_20240509_115906](https://github.com/linshuqin329/UPS-Lite/assets/88111493/7da7f32c-7d5c-41ab-b1da-18375cd90cea)

**Comment by xt2six** [comment]
![img_0685](https://github.com/linshuqin329/UPS-Lite/assets/88111493/59856b7c-c1cd-4070-b6af-94257be3e293)

Good evening linshuqin329, thank you for these details. 

I finally opted to use a terminal block but it is not practical to manage an 18650 battery holder which is larger than the raspberry pi zero. So I gave up and bought another card which doesn't really meet my needs but it provides an 18650 battery slot (GEEKWORM X306 - UPS ; but why a hub USB ... and no RJ45..., no RTC...)

Thanks for support !

**Comment by xt2six** [comment]
![img_0685](https://github.com/linshuqin329/UPS-Lite/assets/88111493/59856b7c-c1cd-4070-b6af-94257be3e293)

Good evening linshuqin329, thank you for these details. 

I finally opted to use a terminal block but it is not practical to manage an 18650 battery holder which is larger than the raspberry pi zero. So I gave up and bought another card which doesn't really meet my needs but it provides an 18650 battery slot (GEEKWORM X306 - UPS ; but why a hub USB ... and no RJ45..., no RTC...)

Thanks for support !
